runelite-client/src/main/java/net/runelite/client/plugins/fightcave/FightCaveOverlay.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, Devin French <https://github.com/devinfrench>
```

Comment:
Do not remove copyright unless you are the original copyright holder. You add copyright.
It appears that git is just being git and this is actually a completely new file with the old name. Maybe renaming this file to `FightCaveWaveOverlay` would be good to clear up confusion and facilitate future expansion.